example/wrangler.toml

```diff
@@ -1,9 +1,9 @@
-name = "axum-cloudflare-adapter-example"
+workers_dev = true
+
 main = "build/worker/shim.mjs"
-compatibility_date = "2022-01-20"
```

Comment:
Looks like we need this back, was there a reason it needed to be removed?

Reply:
But yes, for some unknown reason to me its breaking the build, so I placed it back in.

Sorry for that.